app/models/vm_or_template.rb

```diff
@@ -531,12 +530,6 @@ def genealogy_parent=(parent)
     @genealogy_parent_object = parent
   end
 
-  def save_genealogy_information
-    if defined?(@genealogy_parent_object) && @genealogy_parent_object
-      with_relationship_type('genealogy') { self.parent = @genealogy_parent_object }
```

Comment:
keep in mind. there is now a slight subtlety in this method.

before - encouraged to set this AFTER the record is saved. setting this does not need a `save!`

after - encouraged to set this BEFORE the record is saved. setting this NEEDS `save!` / `update()`